packages/cmr-client/API.md

```diff
@@ -12,14 +12,6 @@ CMR search, without loading them all into memory at once.  Handles paging.</p>
 </dd>
 </dl>
 
-## Functions
```

Comment:
This is interesting to me given that there weren't any code changes to this package in between the versions... I'm inclined to trust it, but still

Reply:
Indeed - it wasn't updated in a prior release.